### PR TITLE
language: remove verbose option

### PR DESCRIPTION
**Pull request by callmehiphop**
Closes https://github.com/GoogleCloudPlatform/google-cloud-node/issues/1782

---

#### TODOs

- [x] Docs
- [x] Tests
  - [x] System
  - [x] Unit

### CODE REVIEW

**Comment by callmehiphop** [comment]
@stephenplusplus before I go in and do it, my plan for the docs was to remove a lot of the params detailing the objects we return to the user in favor of just linking them to the upstream docs. Using `document.annotate()` as [an example](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/packages/language/src/document.js#L251), I'm not sure we need to document this object so heavily if it doesn't really vary from the raw API response too much.

WDYT?

**Comment by stephenplusplus** [comment]
I think if we're returning the raw API response, linking is good. But, if we do any re-mapping, we have to document it.

However, I think @jgeewax wanted to kill the grouping of types, which I *think* means we are only ever returning the raw API response. So, no special docs would be needed, if we do that.

**Comment by callmehiphop** [comment]
Got it, could you PTAL at my latest changes to see if what we'll now be providing in the callback looks ok? If so, I'm going to get started on tests and docs.

**Comment by stephenplusplus** [comment]
LGTM.

**Comment by callmehiphop** [comment]
@stephenplusplus I think I'm done here if you would kindly take a look!

File: packages/language/system-test/language.js [hunk 1]
```diff
@@ -339,66 +310,20 @@ describe('Language', function() {
...
-  function validateAnnotationVerbose(callback) {
+  function validateAnnotation(callback) {
     return function(err, annotation, apiResponse) {
       try {
         assert.ifError(err);
 
         assert.strictEqual(annotation.language, 'en');
 
         assert(is.object(annotation.sentiment));
+        assert(is.number(annotation.sentiment.score));
```
Comment (stephenplusplus) [inline packages/language/system-test/language.js]: For these, let's just check that `annotation` === apiResponse.sentiment` (or whatever). Keep anything that checks that the correct amount of results are returned, but type-checking isn't necessary anymore.

**Comment by stephenplusplus** [comment]
All looks good to me, just the one thing about simplifying the system test checks now that we're just passing back the raw response.

File: packages/language/system-test/language.js (hunk 1 above)
Comment (callmehiphop) [inline packages/language/system-test/language.js]: Done!

Comment (stephenplusplus) [inline packages/language/system-test/language.js]: Woo!

**Pull request merged**